views/importing/queue_items_grouped_album.html

```
@@ -125,6 +125,50 @@ <h4 class="text-sm font-medium text-gray-900 dark:text-white">{{.Track.Title}}</
+                            <i class="fas fa-music fa-xs"></i> New
+                        </button>
+                        <button _="on click toggle .hidden on #pa-lib-{{.ID}}"
+                            class="inline-flex items-center gap-1.5 px-2 py-0.5 rounded text-xs font-medium bg-blue-500/10 border border-blue-400/30 text-blue-600 dark:text-blue-300 hover:bg-blue-500/20 transition-colors">
+                            <i class="fas fa-music fa-xs"></i> Existing
+                        </button>
+                    </div>
+                    <div id="pa-queue-{{.ID}}" class="audio-player hidden flex items-center gap-2 bg-violet-500/5 border border-violet-400/20 rounded-lg px-2.5 py-1.5 w-full">
+                        <button onclick="toggleAudioPlayer(this)" class="play-btn flex-shrink-0 w-5 h-5 flex items-center justify-center rounded-full bg-violet-500/20 hover:bg-violet-500/40 text-violet-600 dark:text-violet-300 transition-colors">
+                            <i class="fas fa-play fa-xs"></i>
+                        </button>
+                        <input type="range" oninput="seekAudioProgress(this)" class="flex-1 h-1 rounded-full cursor-pointer accent-[#463371] mx-1" value="0" min="0" max="100" step="0.1">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)